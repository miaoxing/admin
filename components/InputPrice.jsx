import {InputNumber} from 'antd';

export default (props) => {
  return (
    <InputNumber min={0.01} max={1000000000} precision={2} controls={false} {...props}/>
  );
};
