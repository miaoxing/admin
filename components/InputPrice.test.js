import InputPrice from './InputPrice';
import {render} from '@testing-library/react';

describe('InputPrice', () => {
  test('default', () => {
    const {container} = render(<InputPrice/>);

    expect(container).toMatchSnapshot();
  });
});
