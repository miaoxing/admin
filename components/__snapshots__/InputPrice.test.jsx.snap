// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputPrice default 1`] = `
<div>
  <div
    class="ant-input-number ant-input-number-outlined"
  >
    <div
      class="ant-input-number-input-wrap"
    >
      <input
        aria-valuemax="1000000000"
        aria-valuemin="0.01"
        autocomplete="off"
        class="ant-input-number-input"
        role="spinbutton"
        step="1"
        value=""
      />
    </div>
  </div>
</div>
`;
